hack/e2e.go

```diff
@@ -184,13 +184,14 @@ func TearDown() {
 	runBash("teardown", "test-teardown")
 }
 
+// Up brings an e2e cluster up, recreating it if one is already running.
 func Up() bool {
-	if !tryUp() {
-		log.Printf("kube-up failed; will tear down and retry. (Possibly your cluster was in some partially created state?)")
+	if isUp() {
+		log.Printf("e2e cluster already running; will teardown")
```

Comment:
Only one suggestion-- perhaps should fail if this TearDown fails, rather than calling up again...